src/app/zap-templates/templates/app/af-structs.zapt

```diff
@@ -13,7 +13,7 @@
 // Struct for {{label}}
 typedef struct _{{asType label}} {
 {{#zcl_struct_items}}
-{{ident}}{{asChipUnderlyingType label type}} {{asSymbol label}};
+{{ident}}{{asUnderlyingZclType type}} {{asSymbol label}};
```

Comment:
Is this why the generated types change from `chip::....` to raw integers?

Reply:
Yes. That was it. I have reverted this change.